templates/common/aws/units/afterburn-hostname.service.yaml

```diff
@@ -0,0 +1,18 @@
+name: afterburn-hostname.service
```

Comment:
Also please name this `aws-hostname.service` or so similarly to `gcp-hostname.service`.